### PR TITLE
feat(electron) setup electron integration

### PR DESCRIPTION
**Pull request by michaeljota**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

new feature

* **What is the current behavior?** (You can also link to an open issue here)

no electron integration

* **What is the new behavior (if this is a feature change)?**

electron integration complete. With apropiate npm `scripts` and livereload.

* **Other information**:

`build:electron`: Alias to `build:electron:full`
`build:electron:full`: Build of both part of the Electron application.
`build:electron:renderer`: Build the renderer part of the Electron application.
`build:electron:main`: Build the main part of the Electron application.
`electron`: Bind to Electron
`electron:start`: Start the Electron application from `/dist` 
`postbuild:electron:full`: Call `electron:start`
`prebuild:electron:full`: Call `clean:dist`
`watch:electron`: Start webpack in watch mode with Electron auto reload.

### CODE REVIEW

File: config/webpack.browser.common.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ module.exports = function(options) {
        *
        * See: http://webpack.github.io/docs/configuration.html#resolve-extensions
        */
-      extensions: ['', '.ts', '.js', '.json'],
+      extensions: ['', '.ts', '.js', '.css', '.scss', '.json'],
```
Comment (katallaxie) [inline config/webpack.browser.common.js]: I would not make `.css` and `.scss` to be be resolved.

**Comment by katallaxie** [comment]
👍 seems to be a lot of work. I don't know if we should include this in the starter. It should be very slim and focused on just get dev starting.

**Comment by d3viant0ne** [comment]
Agree with @katallaxie - This doesn't belong in a `starter` repo. Electron, although something I really like is situational at best. This is a great idea but just not something that belongs in this repo.

My suggestion would be to go the route that @NathanWalker with https://github.com/NathanWalker/angular-seed-advanced

Fork the starter, make your additions & squash them into a single commit to make it easier to pick up changes from upstream and once it's in good shape, I'm sure @gdi2290 wouldn't have an issue with adding a link to the README.md to make it easier to find your version of the `advanced starter`

**Comment by michaeljota** [comment]
Ok. Thanks. I will. (Like, really... :P). I'll ping you in a edit here to let you know when I upload it.

**Pull request closed without merging**